cmd/skaffold/skaffold.go

```diff
@@ -38,7 +38,8 @@ func main() {
 		if errors.Is(err, context.Canceled) {
 			logrus.Debugln("ignore error since context is cancelled:", err)
 		} else {
-			color.Red.Fprintln(os.Stderr, err)
+			errOut := color.SetupColors(os.Stderr, color.DefaultColorCode, false)
```

Comment:
Could we not just do the color setup entirely here in `main()`?  It seems odd to have a special case here for the error condition.

Reply:
that's what I was hoping to do, but I realized that we allow some color setup using cli flags, so I'm not sure if that would be possible

Reply:
Oh!  That's worth commenting either here or in main().